packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -128,15 +134,15 @@ class MediaQueryData {
   /// When a mobile device's keyboard is visible `viewInsets.bottom`
   /// corresponds to the top of the keyboard.
   ///
-  /// This value is independent of the [padding]: both values are
-  /// measured from the edges of the [MediaQuery] widget's bounds. The
+  /// This value is independent of the [padding] and [viewPadding]: their values
```

Comment:
I think this is true, but maybe could be clarified by saying that this value is not derived from padding and viewPadding.